### PR TITLE
Add cli ranges comment

### PR DESCRIPTION
**Pull request by pmatulis**
Lots of changes that needed to be made.

### CODE REVIEW

File: en/manage-cli-common.md [hunk 1]
```diff
@@ -65,25 +65,27 @@ Multiple `add=` arguments (and their values) can be used to apply a tag to
...
 
 
-## Reserve IP addresses
+## Create a reserved IP range
 
-To reserve a range of dynamic IP addresses that will be used by MAAS for
-node enlistment and commissioning:
+See
+[Concepts and terms](intro-concepts.md#ip-ranges)
+for an explination of the two kinds of reserved IP ranges MAAS uses.
```
Comment (degville) [inline en/manage-cli-common.md]: typo (explanation)

File: en/installconfig-subnets-dhcp.md [hunk 2]
```diff
@@ -3,36 +3,47 @@ Title: DHCP
...
+commission nodes.**
+
 
 ## Competing DHCP
 
 Enabling your own DHCP server that competes with one that's being managed by
 MAAS can cause serious disruption. Make sure you understand the implications of
 running a DHCP server before doing this. If MAAS detects external DHCP servers
-on its networks, it will show them on the rack controller page in the web UI.
+on its networks, it will display them on the rack controller page in the web UI.
```
Comment (pmatulis) [inline en/installconfig-subnets-dhcp.md]: Agreed. Will fix.

File: en/installconfig-subnets-ipranges.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+- Delete a range
+
+All three actions require administrative privileges and begin by accessing the
+'Networks' page, selecting the desired subnet, and scrolling down to the
+'Reserved' section.
+
+
+## Create a range
+
+Choose 'Reserved range' or 'Reserved dynamic range'. If the latter is chosen,
```
Comment (pmatulis) [inline en/installconfig-subnets-ipranges.md]: Nice. Will change.

File: en/manage-cli-common.md (hunk 1 above)
Comment (pmatulis) [inline en/manage-cli-common.md]: Thank you.

**Comment by degville** [comment]
Looks great - thanks Peter!

**Pull request merged**